src/routes/Apps/App/Testing.tsx

```diff
@@ -481,6 +503,11 @@ class Testing extends React.Component<Props, ComponentState> {
         return ''
     }
 
+    // Progress count is less jumpy when smoothed between in progress and completed
+    displayCount(): number {
+        return Math.round((this.state.doneCount + this.state.completedTestIds.length) * 0.5)
```

Comment:
I didn't understand what the "smoothing" part is.  If `doneCount` and `completedTestIds` are both positive integers the number would always be N + 0.5 which would up. Don't understand the need for `rouding`. Why not just `ceil` or nothing?

Reply:
Done lags completed.  So display will immediately jump.  This averages the two and rounds to a whole number which smooths the display

Reply:
Oh, I see. I read `* 0.5` as `+ 0.5` which is why I was confused. I usually think of averages as dividing by total count of items in series such as `(x + y) / 2`.